Gemfile

```diff
@@ -24,18 +26,15 @@ gem "openssl"
 
 gem "deface"
 
-# if deploying to a dedicated server
-# gem "daemons"
-# gem "delayed_job_active_record"
-# gem "whenever"
```

Comment:
this gems must be uncommented

Reply:
All these three gems are already in the Gemfile. This was duplicates